tests/int.rs

```diff
@@ -0,0 +1,64 @@
+#![cfg(feature = "serde")]
+
+use js_int::{int, Int};
+use serde::{de::IntoDeserializer, Deserialize};
+use serde_test::{assert_ser_tokens, Token};
+
+#[test]
+fn serialize_int() {
+    assert_serialize(100);
+    assert_serialize(0);
+    assert_serialize(-100);
+}
+
+fn assert_serialize(number: i32) {
```

Comment:
I was tempted to use `#[track_caller]` here but then I remembered that the MSRV is 1.35.

Using a macro would also be possible but I don't think it's really worth it.